### ISSUE TITLE
`export-model` generating incorrect NN

### ISSUE THREAD

**Issue by alecgunny** [ML4GW/aframe#390]
A recent run of `export-model` from `main` pointed at a trained network seems to produce a functionally random NN. Here's a comparison of inference on a background event performed using various combinations of a preprocessor and and NN hosted either locally with Torch or with Triton:

![image](https://github.com/ML4GW/aframe/assets/14932242/fd8fb63b-e635-4714-bf83-95075f71ae1d)

The lower sets of output timeseries are what I would expect (and since they're performed with Torch locally, I take them as ground truth), but the upper timeseries show that no matter how the data is preprocessed, the Triton-hosted NN produces nonsense outputs.

By contrast, here is an output timeseries produced by the same model hosted on Triton, but with export done from [this branch](https://github.com/alecgunny/bbhnet/tree/sandbox-environment)

![image](https://github.com/ML4GW/aframe/assets/14932242/9ad0d41a-7933-49f2-83dc-3fbbdb538243)

Clearly, something funky is happening during NN export in `main`. I suspect this may be somehow related to changes made in #376 , but I will do a finer-grained commit search later today to confirm.

**Comment by wbenoit26** [comment]
Looks like there's a difference [here](https://github.com/alecgunny/BBHNet/blob/4f7c862e69c6694ed4bae4bb58a6720a1ae51938/projects/sandbox/export/export/main.py#L134). `main` has `kernel_length = 1.5`, whereas your branch has `kernel_size = 2`. Not sure which the network was trained with though.

**Comment by alecgunny** [comment]
Ok a more exhaustive search reveals that things broke between [8fd65](https://github.com/ML4GW/aframe/commit/8fd65ff870d809828dac3fd567dfbb5b647f2fb6) and [7f55b5](https://github.com/ML4GW/aframe/commit/7f55b5fd11603d5550fe84512b713d8a5c0a7d33). A quick look at their [comparison](https://github.com/ML4GW/aframe/compare/8fd65..7f55b5) shows that the only relevant changes between these are in the `architecture` lib. We changed the class name in the `get_norm_layer` wrapper to `NormLayer` but still returned `GroupNorm`. My model was trained _without_ this bug, and so `norm_groups` was getting passed to the layer as expected. But once I tried to do inference from `main`, this bug came into play and `norm_groups` was functionally be set to `None` (since the wrapped `NormLayer` layer wasn't getting used).

This ended up being a pretty simple fix and is implemented in #391
